keps/sig-storage/3751-volume-attributes-class/README.md

```diff
@@ -694,10 +694,7 @@ VolumeAttributesClass parameters can be considered as best-effort parameters, th
 
 * Basic unit tests for performance and quota system.
 * API conformance tests
-* E2E tests with happy tests in the [K8s storage framework](https://github.com/kubernetes/kubernetes/tree/master/test/e2e/storage/testsuites) for different drivers testing
-* E2E tests using mock driver to cause failure on create, update and recovering cases
-  * [K8s storage framework](https://github.com/kubernetes/kubernetes/tree/master/test/e2e/storage/testsuites)
-  * [csi-tes](https://github.com/kubernetes-csi/csi-test)
+* E2E tests: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/volumeattributesclass.go
 * Test coverage of quota usage with ResourceQuota and LimitRange
 * Measure latency impact to CreateVolume during beta and provide feedback to operators
 * Upgrade and rollback test when the feature gate changes to beta
```

Comment:
Has anyone used/tested quota features of this KEP before we move the whole KEP to GA? This seems like a problem.

Reply:
The latest quota change is this: https://github.com/kubernetes/kubernetes/pull/124360

But it did not merged in yet.

Reply:
sure. But it seems weird to merge that code directly with GA feature-gate when it was never tested/merged before.